src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -171,7 +176,7 @@ def _validate_processed_arg_dicts(self, arg_dicts: list):
         warnings = []
 
         for d in arg_dicts:
-            if not isinstance(d["values_df"], (DataFrame, Callable)):
+            if not isinstance(d["values_df"], (DataFrame, Callable)):  # type: ignore
```

Comment:
Regarding your frustration with checking if something is callable yesterday - are you sure this works as intended?